e3/anod/spec.py

```diff
@@ -144,7 +144,10 @@ def __init__(self, qualifier, kind, jobs=1, env=None):
         :raise: SpecError
         """
         self.deps = OrderedDict()
-        """:type: OrderedDict[e3.anod.deps.Dependency]"""
+        """:type: OrderedDict[str, e3.anod.Anod]"""
+
+        self.dependencies = OrderedDict()
```

Comment:
This is a significant change that change the API. Either we dump the API number and document the change or we try to move this info somewhere else.